mobify/capturejs
# Fix for ADJS-92: iOS 8 Smart Banners - Duplicate banner

Removes the smart banner on iOS 8 devices only when it appears above the tag. 

Status: **Opened for visibility**
Reviewers: **@tedtate @donnielrt @scalvert @jansepar**
JIRA: **https://mobify.atlassian.net/browse/ADJS-92**

## Changes
- Remove smart banner meta tag from head when it is above the mobify tag
- Added examples for testing

## How to test-drive this PR
- Checkout this branch
- `grunt serve`
- view /tests/fixtures/smart-banner-below-tag.html on an iOS8 device
- view /tests/fixtures/smart-banner-above-tag.html on an iOS8 device
- A smart banner should appear for both examples
  
## Notes
- Still needs qunit tests.
